src/org/opendatakit/briefcase/model/BriefcaseAnalytics.java

```diff
@@ -0,0 +1,182 @@
+package org.opendatakit.briefcase.model;
+
+import com.brsanthu.googleanalytics.EventHit;
+import com.brsanthu.googleanalytics.GoogleAnalytics;
+import com.brsanthu.googleanalytics.GoogleAnalyticsConfig;
+import org.opendatakit.briefcase.buildconfig.BuildConfig;
+
+/**
+ *  This class is used to send usage information to ODK's Google Analytics account.
+ *
+ * <P>A library built by 'brsanthu' is utilized to streamline the
+ * process of contacting Google's Measurement Protocol API.</P>
+ * <P>This class also implements the Singleton pattern.</P>
+ *
+ * @author Joao Paulo Knox
+ * @see <a href="https://github.com/brsanthu/google-analytics-java">google-analytics-java</a>
+ */
+public class BriefcaseAnalytics {
+
+  private static final String GOOGLE_TRACKING_ID = BriefcasePreferences.GOOGLE_TRACKING_ID;
+  private static final String UNIQUE_USER_ID = BriefcasePreferences.getUniqueUserID();
+  private static final String EVENT_CATEGORY = "ODK Briefcase";
+
+  private static BriefcaseAnalytics instance = null;
+  private static GoogleAnalytics googleAnalytics = null;
+  private static GoogleAnalyticsConfig config = null;
+  private static Boolean briefcaseTrackingConsent = null;
```

Comment:
I'm still not sure it will stop a second startup being tracked if I remove the field and call it twice. The Boolean variable 'hasTrackedStartup' must have its state persisted across each method call in order to stop a developer accidentally tracking unnecessary startup events.